.github/workflows/ci.yml

```diff
@@ -13,6 +13,7 @@ env:
   APP_ID: 1
   REPO_NAME_SLUG: gpswapui
   PR_NUMBER: ${{ github.event.number }}
+  REACT_APP_SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
```

Comment:
Have you added that to repo settings already?

Reply:
yep! 👌